controller/api/util/api_utils.go

```diff
@@ -220,6 +227,32 @@ func BuildStatSummaryRequest(p StatsSummaryRequestParams) (*pb.StatSummaryReques
 	return statRequest, nil
 }
 
+// BuildEdgesRequest builds a Public API EdgesRequest from a
+// EdgesRequestParams.
+func BuildEdgesRequest(p EdgesRequestParams) (*pb.EdgesRequest, error) {
```

Comment:
I was mirroring the format of `stat.go` as I built this file so that's why I put `BuildEdgesRequest` in `api_utils.go`. I haven't looked into the web implementation yet. Would it make more sense to have `BuildEdgesRequest` in `edges.go`?